src/rpc/server_util.cpp

```diff
@@ -129,3 +133,18 @@ AddrMan& EnsureAnyAddrman(const std::any& context)
 {
     return EnsureAddrman(EnsureAnyNodeContext(context));
 }
+
+void NextEmptyBlockIndex(CBlockIndex& tip, const Consensus::Params& consensusParams, CBlockIndex& next_index)
```

Comment:
In "rpc: add next to getmininginfo" cf0a62878be214cd4ec779aab214221b27b769b6

You don't need to update this, just thinking out loud: would it be better for this to simply be `NextBlockHeader` and to modify `GetDifficulty` and `GetTarget` to accept the block header since the info they need are already in the header.

Reply:
A `CHeader` does not contain `nHeight`, though the height is trivial anyway.

I'm a bit reluctant to expand the scope of this PR even further by refactoring `GetDifficulty`. So I prefer to leave this as a potential followup.